portability-types-common/src/main/java/org/datatransferproject/types/common/models/videos/VideoModel.java

```diff
@@ -90,36 +93,46 @@ public String getName() {
     return super.getName();
   }
 
+  public Date getUploadedTime() {
```

Comment:
Do we use any @Nullable annotations elsewhere? Can help IDEs pick up NPEs and similar.